apps/client/src/features/rundown/time-input-flow/TimeInputFlow.tsx

```diff
@@ -45,7 +46,7 @@ function TimeInputFlow(props: EventBlockTimerProps) {
   };
 
   const warnings = [];
-  if (timeStart > timeEnd) {
+  if (timeStart + duration > dayInMs) {
```

Comment:
I am not sure this is the same thing?

The previous logic said that if a time is scheduled like
22:00 - 09:00 , we say it crosses midnight. but it is not true that the event will have more than 1 day duration

Reply:
in your example 
timeStart: 22h
timeEnd: 9h
duration: 11h

so the calculation 
timeStart + duration > dayInMs
           22 + 11 = 33  > 24h  

so it crossed midnight
the change is so that an event that ends exactly at midnight dose not count as crossing midnight

Reply:
👍